src/s1reader/s1_burst_slc.py

```diff
@@ -447,10 +447,27 @@ def sensing_stop(self):
         d_seconds = (self.length - 1) * self.azimuth_time_interval
         return self.sensing_start + datetime.timedelta(seconds=d_seconds)
 
+    @property
+    def burst_duration(self):
+        '''Returns burst sensing duration as float in seconds.
+
+        Returns:
+        --------
+        _ : float
+            Burst sensing duration as float in seconds.
+        '''
+        return self.azimuth_time_interval * self.shape[0]
```

Comment:
```suggestion
        return self.azimuth_time_interval * self.length
```
Not 100% on the following: I think a name is clearer than indexing a tuple.